src/components/Card/Card.jsx

```diff
@@ -80,7 +81,7 @@ class Card extends React.Component {
           { (title || actions) &&
             <div className={style.top}>
               { title &&
-                <Header h3>
+                <Header h4={hasSmallHeader} h3={!hasSmallHeader}>
```

Comment:
added support for a small header.. speaking with Deena, h4 should become the new header size.